www/class/CentreonLDAPAdmin.class.php

```diff
@@ -666,12 +654,13 @@ private function manageContactPasswords($arId)
                     $ldapContactIdList[] = $row['contact_id'];
                 }
                 if (!empty($ldapContactIdList)) {
-                    $contactIds = implode(', ', $ldapContactIdList);
-                    $statement = $this->db->prepare(
-                        "DELETE FROM contact_password WHERE contact_id IN (:contactIds)"
-                    );
-                    $statement->bindValue(':contactIds', $contactIds, \PDO::PARAM_STR);
-                    $statement->execute();
+                    foreach ($ldapContactIdList as $val) {
```

Comment:
Many contacts can be impacted by this loop.
Using a "queryValues" like array can help enhance performance

Reply:
reverted to previous statement